### PR TITLE
about page: display custom fields

### PR DESCRIPTION
**Pull request by jennur**
Closes https://github.com/inveniosoftware/invenio-communities/issues/892

Adds custom fields to the community "About" page.

## Screenshot
Screenshot 2023-03-28 at 10 04 20

### CODE REVIEW

File: invenio_communities/templates/semantic-ui/invenio_communities/details/about/index.html [hunk 1]
```diff
@@ -51,5 +51,44 @@ <h3 class="ui header">{{ _("Awards") }}</h3>
 
       </dl>
     {% endif %}
+
+
+    {% if custom_fields_ui %}
+      {% set custom_fields = community.ui.custom_fields %}
+      {%- for section_cfg in custom_fields_ui %}
+        {% set section_fields = section_cfg.fields %}
+        {% set section_format = section_cfg.section | replace(" ", "-") %}
```
Comment (ntarocco) [inline invenio_communities/templates/semantic-ui/invenio_communities/details/about/index.html:60]: Unused?

File: invenio_communities/templates/semantic-ui/invenio_communities/details/about/index.html [hunk 2]
```diff
@@ -51,5 +51,44 @@ <h3 class="ui header">{{ _("Awards") }}</h3>
...
+
+        <h2>{{ section_cfg.section }}</h2>
+        {% for field_cfg in section_fields %}
+          {% set field_value = custom_fields.get(field_cfg.field) %}
+
+          {% if field_value %}
```
Comment (ntarocco) [inline invenio_communities/templates/semantic-ui/invenio_communities/details/about/index.html:66]: Too much nesting?

File: invenio_communities/templates/semantic-ui/invenio_communities/details/about/index.html [hunk 3]
```diff
@@ -51,5 +51,44 @@ <h3 class="ui header">{{ _("Awards") }}</h3>
...
     {% endif %}
+
+
+    {% if custom_fields_ui %}
+      {% set custom_fields = community.ui.custom_fields %}
+      {%- for section_cfg in custom_fields_ui %}
+        {% set section_fields = section_cfg.fields %}
+        {% set section_format = section_cfg.section | replace(" ", "-") %}
+
+        <h2>{{ section_cfg.section }}</h2>
```
[the remark this answers is not in the document]
Comment (jennur) [reply invenio_communities/templates/semantic-ui/invenio_communities/details/about/index.html:62]: Just forgot to remove the format-variable! Not in use 😄

File: invenio_communities/templates/semantic-ui/invenio_communities/details/about/index.html [hunk 4]
```diff
@@ -51,5 +51,44 @@ <h3 class="ui header">{{ _("Awards") }}</h3>
...
+        {% for field_cfg in section_fields %}
+          {% set field_value = custom_fields.get(field_cfg.field) %}
+
+          {% if field_value %}
+            {% if field_cfg.template %}
+              {% include field_cfg.template %}
+            {% else %}
+              <dt class="ui tiny header">{{ field_cfg.props.label }}</dt>
+              {% if field_value is string %}
```
[the remark this answers is not in the document]
Comment (jennur) [reply invenio_communities/templates/semantic-ui/invenio_communities/details/about/index.html:71]: Thanks!

File: invenio_communities/templates/semantic-ui/invenio_communities/details/about/index.html [hunk 5]
```diff
@@ -8,13 +8,13 @@
 #}
 
 {% extends "invenio_communities/details/base.html" %}
+{% from "invenio_communities/details/macros/custom_fields.html" import list_vocabulary_values, list_string_values, show_custom_field_depending_on_type %}
```
Comment (ppanero) [inline invenio_communities/templates/semantic-ui/invenio_communities/details/about/index.html:11]: not a fan of long names, what about 
```suggestion
{% from "invenio_communities/details/macros/custom_fields.html" import list_vocabulary_values, list_string_values, show_custom_field %}
```

In the end what kind of logic is inside, is abstracted.

Comment (jennur) [reply invenio_communities/templates/semantic-ui/invenio_communities/details/about/index.html]: Updated! :)

**Pull request merged**